eclipse-plugin/eclipse/cc.codechecker.eclipse.plugin/src/cc/codechecker/plugin/config/CodeCheckerContext.java

```diff
@@ -66,6 +81,13 @@ private CodeCheckerContext() {
         jobRunner = new SimpleJobRunner(actionRunner);
     }
 
+    /**
+     * The refresher for Project ReportList View.
```

Comment:
Does the refreshing mean cleaning the cached values and get the results again from the CodeChecker server?